src/components/common/Navigation.js

```diff
@@ -10,6 +10,9 @@ function Navigation() {
         <Menu onClick={({key}) => setCurrent(key)} selectedKeys={[current]} mode="horizontal">
             <Menu.Item key="home"><Link to ="/"/>Home</Menu.Item>
             <Menu.Item key="case-view"><Link to="/case-view"/>Cases</Menu.Item>
+            <Menu.Item key="calandar"><Link to="/calendar" />Calendar</Menu.Item>
```

Comment:
Very clear organization. I'm sure this is just the antd syntax. However, it is very clear and to the point.